app/Http/Controllers/Admin/PostController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Repositories\Post\PostRepository;
+use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use App\Http\Requests\StorePostRequest;
+
+class PostController extends Controller
+{
+    protected $postRepo;
+
+    public function __construct(PostRepository $postRepo)
+    {
+        $this->postRepo = $postRepo;
+    }
+
+    public function index()
+    {
+        $posts = $this->postRepo->getAll();
+        return view('index', ['posts' => $posts]);
+    }
+
+    public function show_posts()
+    {
+        $posts = $this->postRepo->getAllOrderByDESC();
```

Comment:
sửa tên funct lại nha e, hạn chế viết in hoa hết như vậy

Reply:
bổ sung: DESC a nói từ này e nhé

Reply:
Dạ vâng, em hiểu ạ.